flask/app/datasets.py

```diff
@@ -94,6 +95,8 @@ def list_datasets(page: int, limit: int) -> Response:
         # Since this is an elif clause, we know special cases are already handled above.
         # order_by has been restricted to a known list, so we can safely use getattr
         order = getattr(models.Dataset, order_by)
+    elif order_by == "sequencing_id":
+        order = models.Dataset.sequencing_id
```

Comment:
This is a field of the dataset table, so it is already handled by the prior elif by being included in the list and does not need to be explicitly done here.